docs/mine/lotus/split-markets-miners.md

```diff
@@ -48,71 +82,104 @@ At the moment these `subsystems` are designed to be grouped into two distinct ty
+The repository directory is `~/.lotusmarkets`.
+We are using the following `config.toml`:
 
 ```toml
 [API]
+  # Endpoint where the markets process will serve JSON-RPC requests
   ListenAddress = "/ip4/127.0.0.1/tcp/8787/http"
   RemoteListenAddress = "127.0.0.1:8787"
 
 [Libp2p]
+  # Endpoint for libp2p requests (public)
   ListenAddresses = ["/ip4/0.0.0.0/tcp/24001", "/ip6/::/tcp/24001"]
   AnnounceAddresses = ["/ip4/12.34.56.78/tcp/24001"]
 ```
 
-If you intend to run the `mining/sealing/proving` node on the same machine as the `markets` node, make sure that their listeners do not clash. By default the `lotus-miner` API server listens to port 2345, so in the example configuration above, we change the configuration for the `markets` node to listen to `127.0.0.1:8787`.
+If you intend to run the `mining/sealing/proving` node on the same machine as the
+`markets` node, make sure that their `[API]` listener addresses do not clash.
+By default the `lotus-miner` API server listens to port 2345, so in the example
+configuration above for the `markets` process, we change the configuration for the
+`markets` node API server to listen to `127.0.0.1:8787`.
 
-Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it needs to be publicly accessible, so that clients can make storage and retrieval deals with your system.
+Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it
+needs to be publicly accessible so that clients can make storage and retrieval
+deals with your system.
 
-The `[Libp2p]` section on the `mining/sealing/proving` node can be removed because this node will no longer be running a Libp2p node.
+:: info
+The `[Libp2p]` section of the `config.toml` on the `mining/sealing/proving` node
+can be removed because it will no longer be running a Libp2p node, as we explained above.
+::
 
-### 3. Initialising a `markets` service repository
+#### 4. Initialize a `markets` service repository
 
-1. Create authentication tokens for the `markets` node. This is an online operation, so your `lotus-miner` should be running with its updated configuration (in case you move its API to another IP:PORT)
+1. Create authentication tokens that will be used by the `markets` node to make JSON-RPC
+calls to the `mining/sealing/proving` node.
 
 ```shell
 export APISEALER=`lotus-miner auth api-info --perm=admin`
 export APISECTORINDEX=`lotus-miner auth api-info --perm=admin`
 ```
 
-2. Initialise the `market` node. This performs a one-time setup of the markets node. Part of that setup includes updating the `peer id` in the miner actor by submitting a message on chain. This is necessary so that storage and retrieval clients know that this miner's **deal-making** endpoint is now publicly dialable/reachable on a new address (the new `market` node).
-
-Note that `lotus-miner` interacts with one repository or another depending on the `LOTUS_MINER_PATH` environment variable!
```

Comment:
How does this make sense if I'm going to initialize the markets node on a separate machine? In this case, 
LOTUS_MINER_PATH would be somewhere on another machine.